### DISCUSSION TITLE
各位同好，请问如何用wordless做情感分析？

### DISCUSSION THREAD

**Discussion by daviddalao** [BLKSerene/Wordless#36] [Q&A] [answered]
我看setting里有一个情感分析，请问如何用这个分析文本？

**Comment by BLKSerene** [comment, marked as answer]
Concordancer 里 sentiment 一列 每条索引行都会计算情感（如果语料库对应的语种有情感分析支持的话）

**Comment by daviddalao** [comment]
很奇怪，单独使用preview情感分析，有分值。使用concordancer则不显示。
 

| |
david
|
|
***@***.***
|
---- 回复的原邮件 ----
| 发件人 | ***@***.***> |
| 发送日期 | 2024年7月25日 00:11 |
| 收件人 | ***@***.***> |
| 抄送人 | ***@***.***>,
***@***.***> |
| 主题 | Re: [BLKSerene/Wordless] 各位同好，请问如何用wordless做情感分析？ (Discussion #36) |

Closed #36 as resolved.

**Reply by BLKSerene** [reply]
preview 里测试是选的什么语种呢
concordancer 里 sentiment 那一列 显示什么 no language support? 语料库的语种是什么

**Comment by daviddalao** [comment]
| |
david
|
|
***@***.***
|
---- 回复的原邮件 ----
| 发件人 | ***@***.***> |
| 发送日期 | 2024年7月27日 16:19 |
| 收件人 | ***@***.***> |
| 抄送人 | ***@***.***>,
***@***.***> |
| 主题 | Re: [BLKSerene/Wordless] 各位同好，请问如何用wordless做情感分析？ (Discussion #36) |

preview 里测试是选的什么语种呢
concordancer 里 sentiment 那一列 显示什么 no language support? 语料库的语种是什么
